docs/developer-guide/ksqldb-reference/create-connector.md

```diff
@@ -24,6 +24,9 @@ configuration passed in the WITH clause. Some connectors have ksqlDB templates
 that simplify configuring them. For more information, see
 [Natively Supported Connectors](../../concepts/connectors.md#natively-supported-connectors).
 
+If the IF EXISTS clause is present, the statement doesn't fail if the
+type doesn't exist.
+

```

Comment:
Note sure why this talks about _types_ and them _not_ existing. Isn't this about _connectors_ that _do_ exist?


```suggestion
If the `IF NOT EXISTS` clause is present, the statement does nothing if a connector with the supplied name already exists.

```


??

Reply:
Yes , I have changed this , according to the behavior similar `CREATE STREAMS`  though there was no docs for it